internal/config/schema.go

```
@@ -0,0 +1,264 @@
+			return
+		}
+	}
+	parent.Content = append(parent.Content, &yaml.Node{Kind: yaml.ScalarNode, Value: key}, node)
+}
+
+func writeProjectYAMLDocument(configPath string, doc *yaml.Node) error {
+	var output bytes.Buffer
+	encoder := yaml.NewEncoder(&output)
+	encoder.SetIndent(4)
+	if err := encoder.Encode(doc); err != nil {
+		encoder.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `encoder.Close` is not checked

(errcheck)